User-App/package.json

```diff
@@ -6,6 +6,7 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
+    "test-headless": "ng test --watch=false --browsers=ChromeHeadless",
```

Comment:
This is actually required to run test cases for angular since we need to run our cases whenever we receive a pull request or whenever changes are pushed. That is why headless chrome is used. More info here: https://bit.ly/3bsA3m2

Reply:
* Point taken , actually there are no test written for angular yet , but none the less. Add the same for the Notifier-Service & Search-Service. Both are node.js projects.